capnp/Cargo.toml

```diff
@@ -17,11 +17,13 @@ keywords = ["encoding", "protocol", "serialization"]
 [dependencies]
 quickcheck = { version = "1", optional = true }
 
+embedded-io = { version = "0.5.0", default-features = false, optional = true }
+
 [dev-dependencies]
 quickcheck = "1"
 
 [features]
-alloc = []
+alloc = ["embedded-io?/alloc"]
```

Comment:
What does this `?` mean?

Reply:
If the embedded-io feature is enabled, also enable the embedded-io/alloc feature. 

Without the "?", the embedded-io/alloc feature would be enabled if the "alloc" feature was enabled, even when the embedded-io feature was not.